v2/config/manifests/bases/ibm-metrics-operator.clusterserviceversion.yaml

```diff
@@ -201,8 +201,16 @@ spec:
     is a low risk of losing much unreported data.\n- In an airgap environment, MeterReport
     data must be pulled from data-service and uploaded manually using `datactl`. To
     prevent data loss in a disaster scenario, the data-service volumes should be considered
-    in a recovery plan.\n\n### Cluster permission requirements\n\n|API group             |Resources
-    \             |Verbs                                     |\n|----------------------|-----------------------|------------------------------------------|\n|apps
+    in a recovery plan.\n\n### Subscription Config\n\nIt is possible to [configure](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md)
+    how OLM deploys an Operator via the `config` field in the [Subscription](https://github.com/operator-framework/olm-book/blob/master/docs/subscriptions.md)
+    object.\n\nThe IBM Metrics Operator will also read the `config` and append it
+    to the operands. The primary use case is to control scheduling using [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+    and [NodeSelectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).\n\nA
+    limitation is that the `config` elements are only appended to the operands. The
+    elements in the operands are not removed if the `config` is removed from the `Subscripion`.
+    The operand must be modify manually, or deleted and recreated by the controller.\n\n###
```

Comment:
`The operand must be modify manually`
modified

Reply:
corrected